src/modules/CRUD/components/Details/PropertyItem/Type/GeometryType/IconFromCoordinates/IconFromCoordinates.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import geojsonExtent from '@mapbox/geojson-extent';
+
+
+const getCoordinatesFromType = (type, coordinates) => {
+  if (type === 'MultiPolygon') {
+    return coordinates.flat();
+  }
+  if (type === 'LineString') {
+    return [coordinates];
+  }
+  return coordinates;
+};
+
+const IconFromCoordinates = ({ coordinates, type, width, height }) => {
+  if (type.includes('Point')) {
+    return (
+      <svg viewBox={`0 0 ${width} ${height}`} width={width} height={height}>
+        <circle cx={width / 2} cy={height / 2} r="2" fill="currentColor" />
+      </svg>
+    );
+  }
+
+  const [west, south, east, north] = geojsonExtent({ type: 'Point', coordinates });
+
+  const scaleToCanvas = (num, min, max, scaledMax = width, scaledMin = 0) => (
+    (((scaledMax - scaledMin) * (num - min)) / (max - min)) + scaledMin
+  );
+
+  const coordinatesToCanvas = getCoordinatesFromType(type, coordinates).map(coordinate => (
```

Comment:
Canvas or SVG ?

Reply:
At the end, a SVG tag is rendering. I use the term "canvas" because I was thinking as a painter. Maybe you have a better naming suggestion?